modules/test/playwright/pages/account-admin-web/EditAccountWebsitePage.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+export class EditAccountWebsitePage {
+	readonly addWebsiteButton: Locator;
```

Comment:
I think this one is not used (it's not declared in the constructor). Btw for now we can go ahead